content/millennium/r4/care-provision/nutrition-order.md

```diff
@@ -92,7 +92,7 @@ List an individual NutritionOrder by its id:
 
 ### Authorization Types
 
-<%= authorization_types(provider: true, patient: false, system: true) %>
+<%= authorization_types(provider: true, patient: true, system: true) %>
```

Comment:
Patient is not showing in authorization systems. could you verify

Reply:
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/8897881/117253455-dba63e80-ae64-11eb-9954-94aa5cea22fa.png)